crates/biome_js_semantic/src/events.rs

```diff
@@ -155,8 +159,12 @@ pub struct SemanticEventExtractor {
     /// Number of generated scopes
     /// This allows assigning a unique scope id to every scope.
     scope_count: usize,
-    /// At any point this is the set of available bindings and their range in the current scope
+    /// At any point this is the set of available binding names and associated metadata.
+    /// A binding can be associated to two binding names: a value and a type binding.
     bindings: FxHashMap<BindingName, BindingInfo>,
+    /// Number of generated bindings
+    /// This allows assigning a unique binding index to every binding.
```

Comment:
```suggestion
    /// Number of generated bindings.
    // This allows assigning a unique binding index to every binding.
```

It doesn't seem that the second line is documentation, but more something for the developers.